src/scenes/OrgRouter/index.tsx

```diff
@@ -266,7 +266,7 @@ const OrgRouter = ({ showNavBar, setShowNavBar }: OrgRouterProps) => {
             {!isProduction && (
               <Route path={APP_PATHS.OPT_IN} element={<OptInFeaturesView refresh={reloadPreferences} />} />
             )}
-            {/* 
+            {/*
```

Comment:
Should we remove the redirects router?

Reply:
It seems like we can

Reply:
Yes, I think I commented it to test if it was not needed and looks like it's not!